session/src/main/java/uk/gov/di/ipv/cri/common/api/handler/SessionHandler.java

```diff
@@ -57,7 +59,11 @@ public SessionHandler() {
         this.eventProbe = new EventProbe();
         this.auditService =
                 new AuditService(
-                        SqsClient.builder().build(),
+                        SqsClient.builder()
```

Comment:
Is this meant to be here?

Reply:
we added it in to address a code smell as part of this PR - good spot